### ISSUE TITLE
Update doc-Refitting NumPyro models with ArviZ (and xarray)

### ISSUE THREAD

**Issue by roshnaeem** [arviz-devs/arviz#1801]
## Tell us about it
The doc section [Refitting PyStan (2.x) models with ArviZ (and xarray)](https://arviz-devs.github.io/arviz/user_guide/pystan2_refitting_xr_lik.html) and [Refitting NumPyro models with ArviZ (and xarray)](https://arviz-devs.github.io/arviz/user_guide/numpyro_refitting_xr_lik.html) are almost same. The first one has been updated and the latter needs to be updated. 

## Thoughts on implementation

Update the[Refitting NumPyro models with ArviZ (and xarray)](https://arviz-devs.github.io/arviz/user_guide/numpyro_refitting_xr_lik.html) the same way as other has been changed in https://github.com/arviz-devs/arviz/pull/1798.

## Source file
File: https://github.com/arviz-devs/arviz/blob/main/doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

**Comment by Divyateja04** [comment]
Hey, I'm new to contributing to open-source but would like to try this out, is there anything in specific I need to look out for?

**Comment by canyon289** [comment]
@Divyateja04 Youre welcome to try this out! I'll assign it you. Feel free to get started and ask questions

**Comment by roshnaeem** [comment]
Hello @Divyateja04, I have updated the description of related PR  #1798 for your ease. Feel free to ask as many questions as you want.

**Comment by Divyateja04** [comment]
Sure, give me some time and I'll get back to you(as in I'll work on it and reach out to you)

**Comment by Divyateja04** [comment]
Hey, I made some changes and created a PR, but again I don't know how far I was correct in updating contents. So, can you please check it out and sorry if I did anything wrong.

**Comment by roshnaeem** [comment]
Thank you @Divyateja04, let me check.

**Comment by Divyateja04** [comment]
Yeah and if I did some unnecessary changes let me know and I'll change them back to normal again

**Comment by Narend9324** [comment]
hey ill fix this. let me try it
